test/integration/test_delete_ldap_users.py

```
@@ -179,45 +180,29 @@ def test_default_user_not_deleted(self):
         
-        # Try to authenticate with WRONG password (should fail but NOT delete the user)
-        with self.assertRaises(valkey.exceptions.AuthenticationError):
+        # Try to authenticate with any password (should fail and delete the user)
+        with self.assertRaises((valkey.exceptions.AuthenticationError, valkey.exceptions.ResponseError)):
             test_client = valkey.Valkey(
                 host='localhost',
                 port=6379,
-                username='user1',
-                password='wrongpassword',
+                username='removeduser',
+                # noqa: S106
```

Ruff (0.14.10)

Unused `noqa` directive (unused: `S106`)

Remove unused `noqa` directive

(RUF100)